CRD-DTR/Shared/R4/resources/Questionnaire-R4-PractitionerInfo.json

```diff
@@ -43,10 +43,15 @@
                 "language": "text/cql",
                 "expression": "\"BasicClinicalInfoPrepopulation\".EncounterProviderLastName"
               }
+            },
+            {
+               "url" : "http://hl7.org/fhir/StructureDefinition/minLength",
+               "valueInteger": 2
             }
           ],
           "text": "Last Name",
-          "type": "string"
+          "type": "string",
+          "maxLength": 26
```

Comment:
I am wondering if "minLength" works the same way as "maxLength".  Can you try to remove minLength from the extension list and just add it as a property? Thanks.